lib/active_record_view_model.rb

```diff
@@ -482,8 +482,16 @@ def delete_associated(association_name, associated, deserialize_context: self.cl
     end
   end
 
-  def read_association(association_name)
-    association_data = self.class._association_data(association_name)
+  def read_association(association_spec)
```

Comment:
It's either this or passing through `association_name` into `build_*`. Which do you prefer?